AdvLoggerPkg/Library/AdvancedLoggerAccessLib/AdvancedLoggerAccessLib.c

```diff
@@ -136,6 +168,40 @@ FormatPhasePrefix (
   return (UINT16)PhaseStringLen;
 }
 
+/**
+  FormatDebugLevelPrefix
+
+  Adds a debug level indicator to the message being returned.  If debug level is recognized and specified,
+  returns the debug_level prefix in from the AdvMsgEntryPrefix, otherwise raw debug level value is returned.
+
+  @param  MessageBuffer
+  @param  MessageBufferSize
+  @param  Phase
+
+  @retval Number of characters printed
+*/
+STATIC
+UINT16

```

Comment:
Typically, length is represented by `UINTN`.

Reply:
Set this to UINT16 because the final output type of MessageLen in the BlockEntry is UINT16, so keep these UINT16 would be much easier.